thesis-gui/src/services/http-service.tsx

```diff
@@ -1,20 +1,37 @@
-import axios, { AxiosInstance } from "axios";
+import {HttpStatusCode} from "../utils/http-status-code";
 
-class HttpService {
-    private static axiosInstance: AxiosInstance = axios.create({
-        baseURL: 'http://127.0.0.1:8080/api'
-    });
+export const httpService = {
+    request(path: string, method: string, body?: any) {
+        return fetch(
```

Comment:
Where does this change from axios to fetch come from?

Reply:
I couldn't make axios work with credentials in cookies. I don't know why. What's more, fetch is a modern feature and is natively supported by every browser engine.

Reply:
Ok